src/gossip/data.zig

```diff
@@ -183,6 +186,53 @@ pub const SignedGossipData = struct {
         };
     }
 
+    pub fn wallclockPtr(self: *Self) *u64 {
+        return switch (self.data) {
+            .LegacyContactInfo => |*v| {
+                return &v.wallclock;
+            },
+            .Vote => |*v| {
+                return &v[1].wallclock;
+            },
+            .LowestSlot => |*v| {
+                return &v[1].wallclock;
+            },
+            .LegacySnapshotHashes => |*v| {
+                return &v.wallclock;
+            },
+            .AccountsHashes => |*v| {
+                return &v.wallclock;
+            },
+            .EpochSlots => |*v| {
+                return &v[1].wallclock;
+            },
+            .LegacyVersion => |*v| {
+                return &v.wallclock;
+            },
+            .Version => |*v| {
+                return &v.wallclock;
+            },
+            .NodeInstance => |*v| {
+                return &v.wallclock;
+            },
+            .DuplicateShred => |*v| {
+                return &v[1].wallclock;
+            },
+            .SnapshotHashes => |*v| {
+                return &v.wallclock;
+            },
+            .ContactInfo => |*v| {
+                return &v.wallclock;
+            },
+            .RestartLastVotedForkSlots => |*v| {
+                return &v.wallclock;
+            },
+            .RestartHeaviestFork => |*v| {
+                return &v.wallclock;
+            },
```

Comment:
Either we should remove the first return, or:
```suggestion
        return switch (self.data) {
            .LegacyContactInfo => |*v| &v.wallclock,
            .Vote => |*v| &v[1].wallclock,
            .LowestSlot => |*v| &v[1].wallclock,
            .LegacySnapshotHashes => |*v| &v.wallclock,
            .AccountsHashes => |*v| &v.wallclock,
            .EpochSlots => |*v| &v[1].wallclock,
            .LegacyVersion => |*v| &v.wallclock,
            .Version => |*v| &v.wallclock,
            .NodeInstance => |*v| &v.wallclock,
            .DuplicateShred => |*v| &v[1].wallclock,
            .SnapshotHashes => |*v| &v.wallclock,
            .ContactInfo => |*v| &v.wallclock,
            .RestartLastVotedForkSlots => |*v| &v.wallclock,
            .RestartHeaviestFork => |*v| &v.wallclock,
```

Reply:
4af13ce - great catch